### ISSUE TITLE
Bad links in JavaDoc

### ISSUE THREAD

**Issue by jdeisenberg** [tdf/odftoolkit#275]
- Go to `https://tdf.github.io/odftoolkit/api/odfdom/index.html`
- In the search box, search for `Anchor`
- Choose `org.odftoolkit.odfdom.dom.attribute.text.TextAnchorTypeAttribute`
- Get a 404 not found error.

(Tries to go to `https://tdf.github.io/odftoolkit/api/odfdom/undefined/org/odftoolkit/odfdom/dom/attribute/text/TextAnchorTypeAttribute.html`)

This happens for anything that you search for. If you get rid of the "undefined" in the URL, everything works great.

**Comment by svanteschubert** [comment]
Interesting, it seems to be a general JavaDoc issue:
https://stackoverflow.com/questions/52326318/maven-javadoc-search-redirects-to-undefined-url

I have updated to latest JDK 21.0.2 and created the javadoc this time on Windows.
It worked out-of-the-box without changing Maven JavaDoc plugin parameters, so I have copied the generated JavaDoc deliverables to our doc folder (using ./src/site/copy-javadoc.sh).

I will try again later again a SNAPSHOT release (via "mvn deploy"), the first try failed due to a Sonatype server error:
https://oss.sonatype.org/content/repositories/snapshots/org/odftoolkit/odfdom-java/0.13.0-SNAPSHOT/

Thanks again for pointing this out, David!
